tests/integration/test_recipe_filler.py

```diff
@@ -0,0 +1,193 @@
+"""Tests for _data_finder.py."""
+import contextlib
+import os
+import sys
+import shutil
+import tempfile
+
+import pytest
+import yaml
+
+import esmvalcore._config
```

Comment:
There is still a private import from esmvalcore here

Reply:
:man_facepalming: on it now, cheers dude